profiles.yml

```diff
@@ -0,0 +1,25 @@
+# dbt Cloud CLI does not use a profiles.yml file.
+# We need one for sqlfluff to run the dbt templater in CI,
+# which utilizes dbt-core internally.
+
+snowflake:
+  target: dev
+  outputs:
+    dev:
+      type: snowflake
+      account: "{{ env_var('SNOWFLAKE_SANDBOX_ACCOUNT') }}"
+      user: "{{ env_var('SNOWFLAKE_SANDBOX_USER') }}"
+      authenticator: externalbrowser
+      database: analytics
+      warehouse: transforming
+      schema: dbt_winnie
```

Comment:
Should this be a generic name?

Reply:
great question — interested in your thoughts here:
- i would like for this to be generic
- if it is, as the primary developer on this repo for the time being i have to constantly update it and switch it back when pushing and pulling and that feels a bit wack and error prone
- not including a profiles.yml feels bad though, i like to package one in the repo, especially when in this case it's only so sqlfluff can run the dbt templater.

soooo given those competing considerations any ideas?

maybe leave it specific for add more comments around the stuff you should change? something else i haven't thought of?

Reply:
A comment would be fine! Was just checking you hadn’t forgotten to change it as opposed to disliking it.

Reply:
okay i addressed this! i added a heavily commented example profile next to the profile, and expanded the README to explain more about SQLFluff and pre-commit and why there has to be a profile in this project at all. Thanks for the push on this!